javalib/src/main/scala/java/util/Arrays.scala

```diff
@@ -391,110 +326,102 @@ object Arrays {
   @noinline def binarySearch(a: Array[Long], key: Long): Int =
     binarySearchImpl[Long](a, 0, a.length, key, _ < _)
 
-  @noinline
-  def binarySearch(a: Array[Long],
-                   startIndex: Int,
-                   endIndex: Int,
-                   key: Long): Int = {
-    checkRangeIndices(a.length, startIndex, endIndex)
+  @noinline def binarySearch(a: Array[Long],
```

Comment:
There are still a bunch of spurious changes due to `@inline` starting from here. It's not a *big* deal, but for `git blame` purposes it would be nice not to change lines that don't need changing.

Reply:
I believe I have eliminated the spurious changes and understand the decision rule.
I have poured over the log files several times & hope I did not miss anything.